vyper/ast/folding.py

```diff
@@ -24,6 +23,9 @@ def fold(vyper_module: vy_ast.Module) -> None:
         changed_nodes += replace_subscripts(vyper_module)
         changed_nodes += replace_builtin_functions(vyper_module)
 
+    for node in vyper_module.get_children(vy_ast.VariableDecl, {"is_constant": True}):
+        vyper_module.remove_from_body(node)
```

Comment:
i don't think we actually need to remove constant definitions

Reply:
i mean i know they are currently removed, but it's not really necessary

Reply:
What about `implements: interface` statements? They are currently being removed as well.